app/models/repository_host/bitbucket.rb

```diff
@@ -178,20 +178,6 @@ def self.recursive_bitbucket_repos(url, limit = 5)
       end
     end
 
-    def gather_maintenance_stats
-      if repository.host_type != "Bitbucket" || repository.owner_name.blank? || repository.project_name.blank?
-        repository.repository_maintenance_stats.destroy_all
-        return []
-      end
-
-      metrics = []
-
-      metrics << MaintenanceStats::Stats::Bitbucket::CommitsStat.new(repository.retrieve_commits).fetch_stats
```

Comment:
should we remove `Bitbucket::CommitsStat` now/

Reply:
Removed in [`a127983` (#3281)](https://github.com/librariesio/libraries.io/pull/3281/commits/a12798337a5196e7d52de7395f1871e4f1cf7eda)